index.html

```diff
@@ -27,7 +28,11 @@
                     <nav>
                         <a href="#" class="logo">My Logo</a>
                         <i class="menu-icon fa-solid fa-bars"></i>
-                        <div class="right-nav"><span>Portforlio</span><span>About</span><span>Contact</span></div>
+                        <nav class="right-nav">
+                            <span>Portforlio</span>
+                            <span>About</span>
+                            <span>Contact</span>
+                        </nav>
```

Comment:
- [ ] Great work changing it from the `div` to the `nav` tag. However, your implementation is not complete. The first reviewer asked that you use the `a` tag with `href` attribute instead of `span` tags. This way, it would be easy to navigate to other parts of the application and it would increase your accessibility score. 👍 💻